master/getting-started/bare-metal/bare-metal.md

```diff
@@ -476,31 +477,72 @@ outside the cluster:
    normal host endpoint policy).  In other words, if a host endpoint is defined
    but has no pre-DNAT policies that explicitly allow or deny a particular
    incoming packet, that packet is allowed to continue on its way, and will
-   then be accepted or dropped according to workload policy (if it is going to
-   a local workload) or to normal host endpoint policy (if not).
+   then be accepted or dropped according to normal host endpoint policy applied on
+   forward traffic or a workload policy.
+
+## Policy on forwarded traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from
+ local process only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forwarded traffic:
+- Incoming traffic from a host endpoint to a local workload (container/pod/VM).
+- Outgoing traffic from a local workload to a host endpoint.
+- Traffic from one host endpoint that is forwarded to another host endpoint.
+
+By default, `applyOnForward` is `false`. 
+
+Untracked policies and Pre-DNAT policies must have `applyOnForward` set to `true` 
+because they apply to all forwarded traffic. 
+ 
+Policies created before Calico v2.7.0 will have `applyOnForward` set to the 
+default value `false`. This means host endpoint policies and implicit default-deny 
+rules no longer apply to forward traffic by default. If you want a policy to apply 
+to forwarded traffic then you must explicitly set `applyOnForward` to `true`. 
```

Comment:
I think this paragraph is not quite saying the things that it needs to say, which I suggest is this:

**NOTE**: Calico's handling of host endpoint policy has changed, since before Calico v2.7.0, in two ways:

1. It will not apply at all to forwarded traffic, by default.  If you have existing policy and you want it to apply to forwarded traffic, you need to add `applyOnForward: true`.

2. Even with `applyOnForward: true`, the treatment is not quite the same in Calico v2.7.0 as in previous releases, because - once a host endpoint is configured - Calico v2.7.0 allows forwarded traffic through that endpoint by default, whereas previous releases denied forwarded traffic through that endpoint by default.  If you want to maintain the default-deny behavior ...

and then continue with the existing text just below.

(To be clear, I think this should all be inside a Note, as it is all about transition from a previous Calico release.)